backend/backend/game.py

```diff
@@ -154,7 +154,7 @@ def create_game(host):
         conn.close()
 
 
-def get_games():
+def get_games(with_game_status='waiting'):
```

Comment:
`get_games()` is currently being used in `charge_rent.py`, `increment_turn.py`, and `roll_die.py` to find the game the player is in, so as it stands this change will break those functions.

I think it’s probably a good idea to make it default to listing waiting games, but you’ll need to write a new function for the other files to use that takes a player ID and gives back the game that they’re playing in (by making a database query directly).

Another potential issue is that this function won’t allow us to select *all* games, irrespective of state. That might be ok, because we might never need to access all the games, but I’m not certain of it so I wanted to highlight it – what do you think?

Reply:
Ah yes, I forgot about increment_turn.py and roll_die.py. They need to know about playing games to do their job.

I could modify this function to take a default parameter = None, and if the parameter equals None, then all games are retrieved, otherwise, the games with a specified status are retrieved.

Reply:
Ok, that sounds good. I’ll add the other function and change the other files in another pull request.